shared-worker/firebase-worker.js

```diff
@@ -0,0 +1,54 @@
+importScripts('../firebase/firebase-app.js');
+importScripts('../firebase/firebase-firestore.js');
+var config = {
+  // your config here
+};
+
+/** Port used to communicate with the shared worker. */
```

Comment:
Consider giving "the shared worker" a bit more of a clear label / purpose for explanatory reasons.  E.g. "used to communicate with the cross-tab coordination shared worker."